src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -261,22 +261,23 @@ def compute_isi_violations(sorting_analyzer, isi_threshold_ms=1.5, min_isi_ms=0,
     Returns
     -------
     isi_violations_ratio : dict
-        The isi violation ratio described in [Hill]_.
+        The isi violation ratio.
     isi_violation_count : dict
         Number of violations.
 
     Notes
     -----
-    You can interpret an ISI violations ratio value of 0.5 as meaning that contaminating spikes are
-    occurring at roughly half the rate of "true" spikes for that unit.
-    In cases of highly contaminated units, the ISI violations ratio can sometimes be greater than 1.
+    The returned ISI violations ratio measures the approximate fraction of spikes in each
+    unit which are contaminted. This interpretation is good when the ratio is small, and
+    becomes worse as it grows. In cases of highly contaminated units, the ISI  violations
+    ratio can sometimes be greater than 1.
 
     References
     ----------
-    Based on metrics described in [Hill]_
+    Based on metrics originally implemented in [UMS]_
```

Comment:
What is `[UMS]_`?

Reply:
On nice it is a link for the docs very cool, how about `implemented in Ultra Mega Sort [UMS]_`?